assets/js/type-defs/settings.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @typedef {Object} WooCommerceSiteCurrency
+ *
+ * @property {string} code               The ISO code for the currency.
+ * @property {number} precision          The precision (decimal places).
+ * @property {string} symbol             The symbol for the currency (eg '$')
+ * @property {string} symbolPosition     The position for the symbol ('left',
+ *                                       or 'right')
+ * @property {string} decimalSeparator   The string used for the decimal
+ *                                       separator.
+ * @property {string} thousandSeparator The string used for the thousands
```

Comment:
Not: spacing off here

Reply:
👍 I've got another in progress pull that touches this and I'll fix it there.